### PR TITLE
Add iSyntax

### PR DESCRIPTION
**Pull request by JeroenKnoops**
Add iSyntax for Digital Pathology.

https://www.openpathology.philips.com/isyntax/ - SDK for writing, reading and post-processing data in iSyntax format.

### CODE REVIEW

**Comment by MatthewVita** [comment]
Hi @JeroenKnoops, may I see the software license(s) for this software?

Thanks!

**Comment by JeroenKnoops** [comment]
In every file a copyright header is included and that seems to be from the BSD-3-Clause.
https://spdx.org/licenses/BSD-3-Clause.html

I will contact the iSyntax team to get a confirmation.

**Comment by JeroenKnoops** [comment]
"This Pathology iSyntax software license is **limited to copyright**.
So, no license is granted under other forms of IP, e.g. patents.
While there is no separate license document, the header contains the license."

So no BSD-3.

**Comment by MatthewVita** [comment]
@JeroenKnoops Thanks for looking into this.

I also can confirm that the headers in "Compression", "Image Format", and "Post-processing for Image Viewing" include the 3-Clause BSD License.

> So no BSD-3.

I'm not 100% following here. There is a copyright and it's of the correct format: `Copyright <YEAR> <COPYRIGHT HOLDER>` along with the proper license contents. Are you saying that there's an additional license or some weird legal thing going on elsewhere?

Thanks!

**Comment by kakoni** [comment]
@MatthewVita @JeroenKnoops 
So to recap
1) in iSyntax site you can freely download/view isyntax file compression/format/post proc documentation and code samples.
2) Then there is an seperate SDK which is behind registration. EULA says " The license granted hereunder does not include any license, right, power or authority to subject the SDK, in whole or in part, to Open Source Licensing Terms." (https://www.openpathology.philips.com/eula-research)

Correct?

**Comment by MatthewVita** [comment]
Hi @kakoni !

1. This is correct. All assets there appear to open source.
2. This EULA is somewhat confusing (to me, at least). It appears to allow for open source licensing using GPL, LGPL, and many others. The "Disclaimer" is a lightly modified version of BSD-3, which may be a good indicator.

However, I don't want to say for certain that this is open source or not without further discussion. Did you have any difficultly following the EULA?

**Comment by kakoni** [comment]
Getting back to this, summer heat slowed me down;) 
@JeroenKnoops Could you clarify, what is the licence status for OpenPathology SDK?

**Comment by JeroenKnoops** [comment]
The copyright statements are in all source files... That's it.. I don't know more about the license..

f.e. in `CompressionCodeSamples/create8buthuffmantree.m`:
```
% Copyright (c) 2020 Koninklijke Philips N.V. All Rights Reserved.
%
% Subject to the conditions recited below, a free copyright license is hereby
% granted to you for redistribution and use of the Software in source and
% binary forms, with or without modification, provided that the following
% conditions are met:
% - Redistributions of source code must retain the above copyright notice, this
%   copyright license and the following disclaimer.
% - Redistributions in binary form must reproduce the above copyright notice,
%   this copyright license and the following disclaimer in the documentation
%   and/ or other materials provided with the distribution.
% - Neither the name of Koninklijke Philips N.V. nor the names of its
%   subsidiaries may be used to endorse or promote products derived from the
%   Software without specific prior written permission.
%
% THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
% AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
% IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
% ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
% LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
% DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
% OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
% HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
% STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
% IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
% POSSIBILITY OF SUCH DAMAGE.
```

**Comment by MatthewVita** [comment]
@JeroenKnoops The headers inside CompressionCodeSamples ("Compression", "Image Format", and "Post-processing for Image Viewing") indeed include the 3-Clause BSD License. The interest now is to confirm if the SDK itself is open source/free software by reviewing the SDK's [EULA](https://www.openpathology.philips.com/eula-research/).

I'm particularly interested in the "Open Source Restrictions" and  "License restrictions" clauses but I must admit they are hard to parse.

At the end of the day, the major question (for now) is: "is this SDK dual licensed (commercial & open source license)?". I think it's fair to say that the EULA has many commercial elements. Such elements are later concatenated with open source license verbiage/information. That is the point where I get confused.

**Comment by MatthewVita** [comment]
Closing this PR because we don't have enough specific information regarding the licensing of the SDK as discussed above.

Happy to re-open if we have solid information in the future! Thanks.
